app/main/views/questionnaire.py

```diff
@@ -62,9 +62,16 @@ def get_introduction(eq_id, form_type, collection_id):
 @questionnaire_blueprint.route('<location>', methods=["GET"])
 @login_required
 def get_questionnaire(eq_id, form_type, collection_id, location):
-    get_questionnaire_manager(g.schema, g.schema_json).build_state(location, get_answers(current_user))
+    questionnaire_manager = get_questionnaire_manager(g.schema, g.schema_json)
 
-    return _render_template(location, get_questionnaire_manager(g.schema, g.schema_json).state, template='questionnaire')
+    questionnaire_manager.build_state(location, get_answers(current_user))
```

Comment:
Aren't we unnecessarily building state for the interstitial pages? The same question applies to the POST.